styles.js

```diff
@@ -13,4 +13,24 @@ export default createGlobalStyle`
     padding: 0 25px 90px;
     background-color: #fdfbfb;
   }
+
+  .rdp {
+  --rdp-cell-size: 35px;
+  --rdp-caption-font-size: 14px;
+  --rdp-accent-color: #000;
+  --rdp-background-color: #e7edff;
+  --rdp-accent-color-dark: #000;
+  --rdp-background-color-dark: #180270;
+  --rdp-outline: 2px solid var(--rdp-accent-color); /* Outline border for focused elements */
+  --rdp-outline-selected: 3px solid var(--rdp-accent-color); /* Outline border for focused _and_ selected elements */
+  margin: 1em 0.3em 0;
+}
+.formModal {
+    position: fixed;
+    top: 10%;
+  }
+  .goalModal {
+    position: absolute;
+    top: 5%;
+  }
```

Comment:
you should be able to just copy these into the StyledModal?

Reply:
If they differ you can use props to change the behaviour of the modal

Reply:
Hi @djfarly .
I think my brain was a little bit exhausted at this point, so I choose the more simple way. 
I will move this styling into the StyledModal and use props, as you suggested👍